src/tests/baseservices/exceptions/simple/ParallelCrash.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+// Runtime stability in the presence of concurrent fatal errors
+
+public class ParallelCrash
+{
+    private const int ThreadCount = 10;
+
+    private volatile static int _runningThreads;
+
+    public static int Main()
+    {
+        Thread[] threadMap = new Thread[ThreadCount];
+        for (int threadIndex = 0; threadIndex < ThreadCount; threadIndex++)
+        {
+            Thread thread = new Thread(new ThreadStart(CrashInParallel));
+            threadMap[threadIndex] = thread;
+            thread.Start();
+        }
```

Comment:
Since the `threadMap` array is not used anywhere, you could simplify to:
```C#
        for (int threadIndex = ThreadCount; --threadIndex >= 0; )
        {
            new Thread(CrashInParallel).Start();
        }
```

Reply:
Good point, thanks, removed in 4th commit.